jigasi/rootfs/defaults/sip-communicator.properties

```diff
@@ -142,3 +142,5 @@ org.jitsi.jigasi.transcription.SEND_TXT={{ .Env.JIGASI_TRANSCRIBER_SEND_TXT | de
 org.jitsi.jigasi.transcription.RECORD_AUDIO={{ .Env.JIGASI_TRANSCRIBER_RECORD_AUDIO | default "false"}}
 org.jitsi.jigasi.transcription.RECORD_AUDIO_FORMAT=wav
 {{end}}
+
+org.jitsi.jigasi.MUC_SERVICE_ADDRESS=muc.meet.jitsi
```

Comment:
Please use the corresponding env variable here instead of hardcoding the default.

Reply:
I don't know MUC, XMPP or anything related to the technical background of Jitsi service. All I know is the way docker files are created and scripts are used to establish configuration files here. Due to that I don't know if the variable picked in latest commit is the proper one for there are two variables in env.example using this value both looking like eligible candidates here.

Reply:
No worries. This should be XMPP_MUC_DOMAIN, which defaults to `muc.meet.jitsi`

Reply:
So it is.